server/server.js

```diff
@@ -83,9 +86,20 @@ class DataSyncServer {
     this.server.startListening = startListening.bind(this.server)
 
     // Initialize the schema listener for hot reload
-    this.schemaListener = schemaListenerCreator(schemaListenerConfig)
-    this.debouncedOnReceive = _.debounce(this.onReceive, 500).bind(this)
-    this.schemaListener.start(this.debouncedOnReceive)
+    this.schemaListener = schemaListenerCreator(this.serverConfig.schemaListenerConfig)
+    this.debouncedOnSchemaRebuild = _.debounce(this.onSchemaChangedNotification, 500).bind(this)
+    this.schemaListener.start(this.debouncedOnSchemaRebuild)
+  }
+
+  /**
+   * Starts or restarts express app and Apollo server
+   * @param {boolean} restartApp if true, restarts also Express app
```

Comment:
This seems like it's not meant to be here 🤔 Perhaps it was from an earlier implementation. If you can remove that then we're good to go.